### PR TITLE
feat(websocketclient transasctionupdate): initial handshake work

### PR DESCRIPTION
**Pull request by albertolive**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/types/src/types/ConnectionRequest.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type ConnectionRequest = {
```
Comment (zone-live) [inline packages/types/src/types/ConnectionRequest.ts:1]: ConnectRequest and ConnectionRequest, what's the difference? I know that one is the specific object that we get int he qr code flow, but what about the other? Whats its use?

**Review by zone-live — COMMENTED** [review]
Just a few question I have. Ty!

File: packages/institutionalFeatures/src/institutional-features.ts [hunk 2]
```diff
@@ -133,4 +138,34 @@ export class InstitutionalFeaturesController {
...
+    const state = this.store.getState();
+    this.store.updateState({
+      institutionalFeatures: {
+        ...state.institutionalFeatures,
+        connectionRequest: payload,
+      },
+    });
+  }
+
+  setConnectRequests(payload: ConnectRequest): void {
```
[the remark this answers is not in the document]
Comment (albertolive) [reply packages/institutionalFeatures/src/institutional-features.ts:162]: `connectionRequest` is the connection sent by the QR code feature and it's used to decrypt the payload. `connectRequest` is actually used by the extension to show the custody component where we show the list of accounts to add. Does this makes sense to you?

File: packages/types/src/types/ConnectionRequest.ts (hunk 1 above)
Comment (albertolive) [reply packages/types/src/types/ConnectionRequest.ts:1]: `connectRequest` is actually what we have in the extension. The only thing I did was to create the ts type

**Review by zone-live — APPROVED** [review]

**Pull request merged**